src/microlite_esp/app.py

```diff
@@ -37,40 +37,88 @@ def index(req, res):
         DEBUG:<a href='predict'>Predict</a>
     </ul>
 """)
+    
+@app.route('/delete_images')
+def delete_images(req, res):
+    app_manager = AppManager()
+    req.parse_qs()
+    model = req.form.get('model', 'false')
+    class_name = req.form.get('class', 'false')
+    if class_name != 'false' and model != 'false':

```

Comment:
I'm not sure, at this moment, how this deletion form works, but *IMO* this fragment (using such defaults on form) might be troublesome in the future, but I guess it's okay for now.

Reply:
There is an ISSUE with making DELETE type requests in picoweb and so far i was able to manage it that way. However, I do agree that this is far away from doing it clean way, therefore this could be upgraded in the future.